src/main/java/reposense/git/GitClone.java

```diff
@@ -22,28 +23,38 @@
  * Git clone is responsible for cloning a local/remote repository into a new directory.
  */
 public class GitClone {
-
-    public static final String GIT_CLONE_BARE_OPTION = "--bare";
     private static final Logger logger = LogsManager.getLogger(GitClone.class);
 
     /**
-     * Clones a bare repo specified in the {@code repoConfig}.
+     * Runs {@code git clone --bare} command asynchronously to clone a bare repo specified in the {@code config}
+     * into the folder {@code outputFolderName}.
+     *
+     * @return an instance of {@link CommandRunnerProcess} to allow tracking the status of the cloning process.
+     * @throws GitCloneException when exception has occurred while running the command.
```

Comment:
when an error occurs during command execution would be better. The difference between error and exception is like the difference between describing programs and codes.